json-lint/test/resources/schema/inventory/valid/ligatus.test.json

```diff
@@ -7,7 +7,8 @@
   "adServers": {
     "ligatus": {
       "type": "ligatus",
-      "src": "http://a.ligatus.com/?ids=91606&t=js"
+      "src": "http://a.ligatus.com/?ids=91606&t=js",
```

Comment:
I believe we have more properties from ligatus. Let's add them and make a test for it. Same for appnexus

Reply:
Only SRC: https://github.com/ampproject/amphtml/blob/master/ads/ligatus.md

Reply:
👍